src/Recommender/Engagement/ImageTracked.jsx

```diff
@@ -18,14 +25,14 @@ return (
         onClick: state.clicked,
       }}
     />
-      <Widget
-        src="${REPL_MOB}/widget/Image"
-        props={{
-          image: props.profileImage,
-          alt: props.profileImageAlt,
-          fallbackUrl:
-            "https://ipfs.near.social/ipfs/bafkreibiyqabm3kl24gcb2oegb7pmwdi6wwrpui62iwb44l7uomnn3lhbi",
-        }}
-      />
+    <Widget
+      src="${REPL_MOB}/widget/Image"
+      props={{
+        image,
```

Comment:
is the intent here to create this object only once, instead of on each render?

Reply:
This logic was needed to create the correct `image` object for the `Image` component based on the value of `props.profileImage`. Sometimes it's a full URL and other times it's an IPFS CID.